tasmota/tasmota_xsns_sensor/xsns_53_sml.ino

```diff
@@ -1768,6 +1768,15 @@ void SML_Decode(uint8_t index) {
     return;
   }
 
+
+  struct METER_DESC *md = &meter_desc[index];
+  // start of serial source buffer
+  cp = md->sbuff;
+  if (md->flag & 32 && md->type == 's' && *cp != SML_SYNC) {
```

Comment:
Wouldn't the code be easier to understand if a constant with a descriptive name were used for the numerical value 32? And since a bit is obviously being tested here, the use of 0x20 is a good idea.

Reply:
Is this the same comment as https://github.com/arendst/Tasmota/pull/21497#pullrequestreview-2082718853 ?
This is already fixed.